src/components/HoursReportRow/HoursReportRow.js

```diff
@@ -0,0 +1,22 @@
+import './HoursReportRow.css'
+
+import { BsThreeDotsVertical } from 'react-icons/bs';
+
+const HoursReportPage = (props, key) => {
+    const subjects = props.subjects
+    var mystyle = {
+        backgroundColor: props.approval == 1 ? "#a1d47f" : props.approval == 0 ? "#ffd300" : "#ffa1a1"
+    };
+
+    return (
+        <tr style={mystyle} className="c-hours-report-row">
+            <td>{props.date}</td>
+            <td>{props.projectName}</td>
+            <td><select>{subjects.map(subject => <option>{subject.subject}</option>)}</select></td>
```

Comment:
instead of select just print the subject name (like in project). You can use the actionid to get the subject name